src/app/templates/app/submit_new_license.html

```diff
@@ -4,12 +4,15 @@
 {% block head_block %}
     <link href="{% static 'css/editor.css' %}" rel="stylesheet" type="text/css">
     <link href="{% static 'css/editor/diffview.css' %}" rel="stylesheet" type="text/css">
+    <style>
+      .body1 {
+        display: none;
+      }
+    </style>
 {% endblock %}
 
-{% block body1 %}
-    {% include "app/modal.html" %}
-{% endblock %}
```

Comment:
Can't remove modal.html. This modal box is being used in this html page for github login